mava/utils/make_env.py

```diff
@@ -199,6 +215,41 @@ def make_gigastep_env(
     return train_env, eval_env
 
 
+def make_gym_env(
+    config: DictConfig,
+    num_env: int,
+    add_global_state: bool = False,
+) -> gymnasium.vector.AsyncVectorEnv:
+    """
+     Create a gymnasium environment.
+
+    Args:
+        config (Dict): The configuration of the environment.
+        num_env (int) : The number of parallel envs to create.
+        add_global_state (bool): Whether to add the global state to the observation. Default False.
+
+    Returns:
+        Async environments.
+    """
+    base_env_name = config.env.scenario.name
+    env_maker, wrapper = _gym_registry[base_env_name]
+
+    def create_gym_env(config: DictConfig, add_global_state: bool = False) -> Environment:
+        env = env_maker(**config.env.scenario.task_config)
+        wrapped_env = wrapper(env, config.env.use_shared_rewards, add_global_state)
+        if config.env.add_agent_id:
+            wrapped_env = GymAgentIDWrapper(wrapped_env)
+        wrapped_env = GymRecordEpisodeMetrics(wrapped_env)
+        return wrapped_env
+
+    envs = gymnasium.vector.AsyncVectorEnv(
+        [lambda: create_gym_env(config, add_global_state) for _ in range(num_env)],
+        worker=async_multiagent_worker,
+    )
+
+    return envs
+
+
 def make(config: DictConfig, add_global_state: bool = False) -> Tuple[Environment, Environment]:
```

Comment:
This change is a bit tricky since the number of created envs varies depending on whether they are training, evaluation or absolute metric envs. 
Should I add the `num_envs` as a parameter to the `make()` function? it will only be used by the gym envs 
or should I make it return a function that creates the envs?

Reply:
As discussed offline,  please just leave it as is it's confusing to see other Jax make_env having this parameter 🙏